release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -2043,6 +2045,8 @@ The issue is resolved in this release. (link:https://issues.redhat.com/browse/OC
 
 * With this update, the size is reduced for audit log files that are stored in the `kube-apiserver`, `openshift-apiserver`, and `oauth-apiserver` pods for hosted control planes. (link:https://issues.redhat.com/browse/OCPBUGS-31106[*OCPBUGS-31106*])
 
+* Previously, the Hypershift Operator was not using the `RegistryOverrides` mechanism to inspect the image from the internal registry. With this release, the metadata inspector works as expected during the Hypershift Operator reconciliation, and the `OverrideImages` are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-29494[*OCPBUGS-29494*])
```

Comment:
```suggestion
* Previously, the Hypershift Operator was not using the `RegistryOverrides` mechanism to inspect the image from the internal registry. With this release, the metadata inspector works as expected during the Hypershift Operator reconciliation, and the `OverrideImages` mechanism is properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-29494[*OCPBUGS-29494*])
```

Reply:
I have no idea if that is correct I just couldn't find the noun that should follow that anywhere in the repo.